### ISSUE TITLE
Regex expression not working \b

### ISSUE THREAD

**Issue by MakerKath** [sarahkittyy/KeywordTracker#56]
Was trying to use the \bmessage*\b expression, where both a regex tester and sublime text2 says would tag on "message" or "messageeee", but not "Kmessage", "MessageI" and so on.

I found that in KeywordTracker, the plugins just doesn't respond when the \b expression is used. Break specifically when \ is used it seems.

**Comment by sarahkittyy** [comment]
Are you properly adding in the beginning and end `/` to signify regex?

**Comment by sarahkittyy** [comment]
![image](https://user-images.githubusercontent.com/31551546/177448903-f05b6907-1ea9-4960-9be5-7248012af173.png)

**Comment by MakerKath** [comment]
![image](https://user-images.githubusercontent.com/25482230/177460446-4d596686-2b6f-4803-8476-9d3548937c3d.png)
The first few attempts I did without the / at both the beginning and end, that seems to have been it. 
I only noticed it working after I added an additional line where I turned all the \b into /b, then removed the line and it's still working. Only had to relaunch one discord instance since it still wasn't working there.

Thank you ^-^
